### PR TITLE
Change assembly output file extension from .s to .spvasm

### PR DESCRIPTION
**Pull request by Qining**

### CODE REVIEW

File: glslc/src/file_compiler.h [hunk 1]
```diff
@@ -74,7 +74,7 @@ class FileCompiler {
 
   // Sets the flag to indicate disassembly mode. In this mode, the compiler
   // emits disassembled textual output, instead of outputting object files.
-  // This method also sets the output file extension to ".s" and disables
+  // This method also sets the output file extension to ".spvasm" and disables
```
Comment (dekimir) [inline glslc/src/file_compiler.h]: Consider also changing lines 111 and 133.

Comment (dneto0) [inline glslc/src/file_compiler.h]: +1.  Yes need to change those as well.

**Comment by antiagainst** [comment]
LGTM.

**Comment by dneto0** [comment]
+2.  I'll squash and merge

**Pull request merged**